### PR TITLE
Add Homepage for AI Photo Web App

### PR DESCRIPTION
**Pull request by bhydemi**
This PR introduces the homepage for the AI Photo Web App. The homepage showcases the main features of the app, including Style Transfer, Text to Image, and Filter Application, with eye-catching images and concise descriptions. The aim of the homepage is to provide users with a clear and engaging overview of the capabilities of the AI Photo Web App.

This PR includes the following changes:

Addition of the homepage to the app
Implementation of columns to display the main features of the app
Addition of images and descriptions for each of the app's main features
Refinements to the overall design and layout of the homepage.

### CODE REVIEW

File: frontend/image_transform.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import cv2 
+
+def transform_photo(link:str,resize=(480, 480)):
```
[the remark this answers is not in the document]
Comment (bhydemi) [reply frontend/image_transform.py]: I have made adjustements to this typing

File: frontend/image_transform.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    load_img = cv2.imread(link)
```
[the remark this answers is not in the document]
Comment (bhydemi) [reply frontend/image_transform.py:16]: Also added docstrings to this and all files

**Comment by Zaubeerer** [comment]
OK, from my perspective we can now merge it :)

**Comment by bhydemi** [comment]
> OK, from my perspective we can now merge it :)

You have to approve merge

**Review by Zaubeerer — APPROVED** [review]
Looks good to me :)

**Pull request merged**